contracts/UserContract.sol

```diff
@@ -4,15 +4,30 @@ pragma solidity >=0.7.0 <=0.9.0;
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "./types/UserTypes.sol";
 import "./Callable.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
 
 /**
  * @title UserContract
  * @dev This contract work as a centralized user's system, where all users has your userType here
  */
 contract UserContract is Ownable, Callable {
+  
   mapping(address => UserType) internal users;
+  mapping(uint => Delation) private idToDelation;
 
+  using Counters for Counters.Counter;
```

Comment:
Esse `using Counters for Counters.Counter;` vc pode adicionar no início do contrato. Lá na linha `14`.  Só separa ele dos mapping com um espaço